server/sonar-server/src/main/java/org/sonar/server/setting/ThreadLocalSettings.java

```diff
@@ -26,6 +26,8 @@
 import java.util.Objects;
 import java.util.Optional;
 import java.util.Properties;
+import javax.annotation.CheckForNull;
+import org.apache.ibatis.exceptions.PersistenceException;
```

Comment:
Depend on mybatis is not optimal but it does the job... A unit test could be added by mocking SettingLoader.